Tests/web3swiftTests/localTests/EIP1559BlockTests.swift

```diff
@@ -124,8 +124,8 @@ class EIP1559BlockTests: LocalTestCase {
             (Web3.InitialBaseFee, 12_965_000, 20000000, 11000000, 1012500000)              // current above target
         ]
 
-        headerArray.forEach { (touple: (parentBaseFee: BigUInt, parentNumber: BigUInt, parentGasLimit: BigUInt, parentGasUsed: BigUInt, expectedBaseFee: BigUInt)) in
-            let parent = Block(number: touple.parentNumber,
+        headerArray.forEach { (tuple: (parentBaseFee: BigUInt, parentNumber: BigUInt, parentGasLimit: BigUInt, parentGasUsed: BigUInt, expectedBaseFee: BigUInt)) in
+            let parent = Block(number: tuple.parentNumber,
```

Comment:
Please undo this changes, as they affecting code, rather then comments and should be reviewed separately.